src/content/team/team.json

```diff
@@ -73,6 +73,19 @@
         "email": "floody.o@northeastern.edu"
       }
     },
+    {
+      "name": "Zefeng (Daniel) Wang",
+      "team": {
+        "name": "Eboard",
+        "role": "Head of Booling | Dolphin of the East"
```

Comment:
<img width="560" alt="Screen Shot 2022-04-24 at 12 33 46 AM" src="https://user-images.githubusercontent.com/48878676/164956813-5290a433-2974-4cbe-b206-55d376e626a5.png">

Reply:
pls see above comment

Reply:
Did you mean boiling? booing?

Reply:
book-keeping?

Reply:
big-bootying?

Reply:
bowling?